Meadowlark-js/tests/http/local.common.collection.issue.http

```diff
@@ -0,0 +1,168 @@
+#### These are the manual components of the automated system tests
```

Comment:
Full complex resource documents to use to test schema. If they work, they'll fail on reference validation - after schema validation.